### ISSUE TITLE
Fixing CodeCatz image margin in navbar

### ISSUE THREAD

**Issue by anuschka** [CodeCatz/TrackCat#70]
Img style definition in style.img overwriting navbar-brand .img definition in style.css. Do I just move navbar-brand.jpg to the bottom or there is a better way to do it?

**Comment by gandalfar** [comment]
Hmm, can you rephrase the question?

**Comment by anuschka** [comment]
Yes :smile: I would like to fix the CodeCatz picture in the navigation that got out of whack :smile:

**Comment by gandalfar** [comment]
Show a screenshot of a problem, describe what you're seeing in terms of code snippets and we'll suggest something. Initial description isn't giving enough details.

**Comment by anuschka** [comment]
OK :smile_cat:

**Comment by anuschka** [comment]
This is the picture that I would like to fix regarding size and margins. My idea is to move the following code to the bottom of style.css. I am sure there is a better way to resolve the conflict between two same definitions in style.css :smile: 
![brand icon](https://cloud.githubusercontent.com/assets/9576281/5692551/94928124-98fa-11e4-811b-35d66b18262d.jpg)

**Comment by anuschka** [comment]
This is the code I would move down...

/* Fit CodeCatz image in navbar-brand. */
.navbar-brand{
    padding: 0px;
}
.navbar-brand img{
    height: 100%;
}

**Comment by goranche** [comment]
> resolve the conflict between two same definitions in style.css

I'm not sure I understand what you mean by this... :smile: 
but sure, go ahead, fix it, create a PR, and we'll see there

**Comment by gandalfar** [comment]
Don't "move" it down. Fix it at the source, unless it's from bootstrap. In that case, you would declare it again, yes.

**Comment by goranche** [comment]
fixed by #71
